javascript/MaterialXView/source/viewer.js

```diff
@@ -9,7 +9,7 @@ import { RGBELoader } from 'three/examples/jsm/loaders/RGBELoader.js';
 
 import { prepareEnvTexture, getLightRotation, findLights, registerLights, getUniformValues } from './helper.js'
 import { Group } from 'three';
-import { GUI } from 'dat.gui';
```

Comment:
This is the library which is an add-on to Three.js but can directly be pulled in.